client/components/Links.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import cssLinks from './Links.css';
+
+// Component //
+class Links extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const stack = this.props.stack;
+    const git = this.props.git;
```

Comment:
might want to look into `PropTypes` to document what kinds of props a React component takes